examples/rbd/storageclass.yaml

```diff
@@ -162,6 +162,38 @@ parameters:
    # stripeCount: <>
    # (optional) The object size in bytes.
    # objectSize: <>
+
+   # rbd volume QoS.
```

Comment:
why not use https://kubernetes.io/docs/concepts/storage/volume-attributes-classes/#the-volumeattributesclass-api as these values are not immutable as anything mentioned in the SC is mutable?

Reply:
About this:
1. At the beginning of our discussion on this functional requirement, SC was our object of discussion. According to the discussion content, I placed these QoS parameters in SC.
2. In my opinion, VolumeAttributesClass is also nice solution. Existing SC parameters mutable is indeed an advantage. In addition, VolumeAttributesClass need higher k8s version support, SC does not have this requirement.